src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -1786,11 +1785,13 @@ static int _xccdf_policy_generate_fix_kickstart(struct oscap_list *rules_to_fix,
 		"rootpw %s\n"
 		"\n"
 	);
-	char *password = oscap_generate_random_string(24, NULL);
-	char *common = oscap_sprintf(common_template, password);
-	_write_text_to_fd(output_fd, common);
-	free(password);
-	free(common);
+	if (raw == 0) {
+		char *password = oscap_generate_random_string(24, NULL);
+		char *common = oscap_sprintf(common_template, password);
+		_write_text_to_fd(output_fd, common);
```

Comment:
But without the items from the `common_template`, the generated kickstart won't be sufficient for automated installation of the operating system, in other words the installation will require user interaction.

Reply:
The `raw` form is not meant to be easy-consumable, its purpose is to be convenient for parsing and customization using automation tools. It is going to be tools-friendly, not user-friendly.